src/containers/Member.js

```diff
@@ -168,6 +151,27 @@ const MemberPage = ({ releaseBranch = BRANCH.master }) => {
 
 MemberPage.propTypes = {
   releaseBranch: BRANCH_PROP_TYPES,
+  memberData: propTypes.object,
```

Comment:
could we also specify the shape of memberData here

Reply:
fixed